test/cqlpy/test_secondary_index.py

```diff
@@ -2100,3 +2100,17 @@ def test_limit_partition_slice(cql, test_keyspace):
         rs = cql.execute(f'SELECT pk, ck2 FROM {table} WHERE ck1 = 1 LIMIT 3')
         assert sorted(list(rs)) == [(1,1), (1,2), (2,1)]
         assert rs.has_more_pages == False
+
+def test_index_metrics(cql, test_keyspace, scylla_only):
+    with new_test_table(cql, test_keyspace, "p int PRIMARY KEY, v int") as table:
+        index_name = unique_name()
+        cql.execute(f"CREATE INDEX {index_name} ON {table}(v)")
+        wait_for_index(cql, test_keyspace, index_name)
+        initial_metrics = ScyllaMetrics.query(cql)
+        initial_count = initial_metrics.get(f'scylla_index_query_latencies_count', {"idx": index_name, "ks": test_keyspace})
+        if initial_count is None:
```

Comment:
Is this a ScyllaMetrics bug? Since ScyllaMetrics is already asked to sum the metrics from different shards together, shouldn't the sum of zero results be zero, not None?

By we don't have to fix this right now, of course. This is fine.

Reply:
Still, when ScyllaMetrics is asked to **sum** some metric, it should return 0, not "None", when it sums up zero results. But again, no need to fix this now.